### PR TITLE
add 4.14-4.17 cpou jobs

### PR DESCRIPTION
**Pull request by jiajliu**
1. this is another part of [OTA-1196](https://issues.redhat.com//browse/OTA-1196) to add 4.14-4.17 control plane only update jobs.
2. extend the timeout of upgrade-eus-unpause-worker-mcp 

test result:
[azure/arm](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55333/rehearse-55333-periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-4.17-cpou-upgrade-from-4.14-azure-ipi-fullyprivate-proxy-f14/1821730167123349504) succeed.
[nutanix/amd](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55333/rehearse-55333-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-nutanix-ipi-proxy-fips-f14/1822822586749292544) succeed.
[ibmcloud/amd](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55333/rehearse-55333-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-ibmcloud-ipi-f14/1822923878725324800) succeed.

cc @jianlinliu @shellyyang1989 to review, thank you.

### CODE REVIEW

**Comment by jiajliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-ibmcloud-ipi-f14 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-4.17-cpou-upgrade-from-4.14-azure-ipi-fullyprivate-proxy-f14 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-4.17-cpou-upgrade-from-4.14-aws-ipi-sdn-arm-f14

**Comment by jiajliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-nutanix-ipi-proxy-fips-f14 periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-4.17-cpou-upgrade-from-4.14-gcp-ipi-ovn-ipsec-arm-mixarch-f14 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-4.17-cpou-upgrade-from-4.14-baremetal-upi-ovn-ipv4-f14

**Comment by jiajliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-ibmcloud-ipi-f14 periodic-ci-openshift-openshift-tests-private-release-4.17-arm64-nightly-4.17-cpou-upgrade-from-4.14-azure-ipi-fullyprivate-proxy-f14

**Comment by jiajliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-ibmcloud-ipi-f14 periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.14-nutanix-ipi-proxy-fips-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: vsphere-ipi-ovn-dualstack-f14
+  cron: 33 12 9,24 * *
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      FILTERS_ADDITIONAL: ~TELCO&
+      FIPS_ENABLED: "true"
```
Comment (shellyyang1989) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.14.yaml:124]: No `fips` in the profile name.

Comment (jiajliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.14.yaml]: dropped

**Comment by shellyyang1989** [comment]
lgtm

**Comment by jiajliu** [comment]
cc @liangxia to review,thank you.

**Pull request merged**